pkg/controller/daemon/daemon_controller.go

```diff
@@ -1147,6 +1147,7 @@ func (dsc *DaemonSetsController) updateDaemonSetStatus(ctx context.Context, ds *
 			}
 		} else {
 			if scheduled {
+				klog.V(2).Infof("Found misscheduled pod for daemon set %#v on node %s", ds, node.Name)
```

Comment:
Use `InfoS` https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#structured-logging-in-kubernetes

Reply:
Pivoted to `InfoS` with c9da4be37ba -> 867538b697a.  Did you want me to tack on a commit porting the existing log calls in this file while I'm touching things?